Makefile

```diff
@@ -38,6 +38,10 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
```

Comment:
you need to test this. I guess it is still using GPATH 
you need to add go build-mod=vendor